docs/pages/access-controls/guides/dual-authz.mdx

```diff
@@ -41,36 +41,39 @@ two team members for a privileged role `dbadmin`.
 Verify that your Teleport client is connected:
```

Comment:
We should make it clear that this dual authz also work with Slack. I had someone ping me last week if it only worked with Mattermost.

Reply:
Note added